roc-wrapper.go

```diff
@@ -0,0 +1,83 @@
+package roc
```

Comment:
Can we move this file into the `roc` directory?

AFAIK the common convention is to use the same name for the last component of the import path and the package itself. This repo is named `roc-go` so if the import path will be `github.com/roc-project/roc-go` and the imported package will be `roc`, it will be a bit confusing. For that reason I suggest to move the package into the `roc` directory, so that the import path will be `github.com/roc-project/roc-go/roc`.

What do you think?

Reply:
sure, can do. I initially thought that c-for-go will remove the file, but it's not the case.